### PR TITLE
Add functionality to reset password, and removed one useless console log

### PR DESCRIPTION
**Pull request by Hakusen-git**
**Issue:**
The settings form at the moment has no real functionality, I added the amplify method to reset the user's password.

**Solution:**
Using amplify library to achieve changePassword flow, also show a modal to remind user the password has been reset.

**Risk:**
N/A

**Reviewed By:**
[Edit the message as to who reviewed it here]

### CODE REVIEW

File: src/pages/settings.jsx [hunk 1]
```diff
@@ -75,88 +83,160 @@ const useStyles = makeStyles((theme) => ({
 function Settings(props) {
   const classes = useStyles();
   let history = useHistory();
-  const { handleSubmit } = useForm();
+  const [confirmPasswordError, setPasswordConfirmError] = useState(false);
+  const [passwordFormatError, setPasswordFormatError] = useState(false);
+  const [wrongPasswordError, setWrongPasswordError] = useState(false);
+  const { register, handleSubmit, reset } = useForm();
+
+  const [open, setOpen] = React.useState(false);
```
Comment (kcho9906) [inline src/pages/settings.jsx]: just useState

File: src/pages/settings.jsx [hunk 2]
```diff
@@ -75,88 +83,160 @@ const useStyles = makeStyles((theme) => ({
...
+
+    try {
+      const currentUser = await Auth.currentAuthenticatedUser();
+      await Auth.changePassword(currentUser, data.currentPassword, newPassword);
+      setWrongPasswordError(false);
+      reset({});
+      handleClickOpen();
+    } catch (error) {
+      setWrongPasswordError(true);
+      console.log(error);
```
Comment (kcho9906) [inline src/pages/settings.jsx]: Remove console logs

**Review by kcho9906 — DISMISSED** [review]

File: src/pages/settings.jsx (hunk 1 above)
Comment (Hakusen-git) [reply src/pages/settings.jsx]: Updated, removed React.

File: src/pages/settings.jsx (hunk 2 above)
Comment (Hakusen-git) [reply src/pages/settings.jsx]: Removed console log

**Review by kcho9906 — APPROVED** [review]

**Pull request merged**